src/helpers/pivot/table_spreadsheet_pivot.ts

```diff
@@ -305,6 +311,42 @@ export class SpreadsheetPivotTable {
       fieldsType: this.fieldsType,
     };
   }
+
+  sort(
+    measure: string,
+    sortedCol: PivotSortedColumn,
+    getValue: (measure: string, domain: PivotDomain) => FunctionResultObject
+  ) {
+    if (this.isSorted) {
+      return;
+    }
+    const getSortValue = (measure: string, domain: PivotDomain): number => {
+      const rawValue = getValue(measure, domain).value;
+      return typeof rawValue === "number" ? rawValue : -Infinity; // the only non-number values are empty aggregates
```

Comment:
I do not understand this comment :sweat_smile: Is the comment useful ? I feel the code is clear enough